buildbot/osuosl/master/config/builders.py

```diff
@@ -572,15 +573,21 @@
     'builddir': "clang-arm64-windows-msvc-2stage",
     'factory' : ClangBuilder.getClangCMakeBuildFactory(
                     vs="manual",
-                    clean=False,
+                    clean=True,
                     useTwoStage=True,
-                    checkout_flang=True,
                     testStage1=False,
                     extra_cmake_args=[
+                        "-DLLVM_TARGETS_TO_BUILD=AArch64",
```

Comment:
It's obvious in hindsight but if we're not testing stage 1, of course it only needs to be able to produce code for the host. Great thinking!

Every saving helps with this particular build.